docs/installation.rst

```diff
@@ -0,0 +1,40 @@
+.. highlight:: shell
+
+.. IMPORTANT: Update this file with your recommended installation instructions,
+   As an example, two different installation instructions are shown next.
+
+============
+Installation
+============
+
+.. The following installation instructions are recommended if the plugin is uploaded to pypi.
+
+Stable release
+--------------
+
+To install pysteps_nwp_importers, run this command in your terminal:
```

Comment:
Isn't the name of the package "pysteps-nwp-importers"?

Reply:
Correct! I started of with pysteps_nwp_importers and then changed it to be more consistent with the other repositories. I'll change it.